pibooth/view/background.py

```diff
@@ -498,7 +501,7 @@ def resize(self, screen):
                 # Right arrow
                 self.right_arrow = pictures.get_pygame_image(
                     "printer_touch.png", size, hflip=False, vflip=False, color=self._text_color)
-                x = int(self._rect.left + self._rect.width * 0.70
+                x = int(self._rect.left + self._rect.width * 0.80
```

Comment:
Why changing this ratio?

Reply:
I've changed the ratio because the text is too much positioned on the left side. I've made some tests and the users touch the text and in this case, it trigger a print instead of forgotting the photo. But even with this change, the text positionning for touchscreen are not really good. I will perhaps made some refactor of text positionning for the touchscreen version.

Reply:
The probile is for example in this case : 
![image](https://github.com/pibooth/pibooth/assets/13900677/6e016050-1ba8-43f0-891b-2da7802d0e3a)
The users touch the text and trigger printing. I think I will move the text on the left side to be sure than when users touch the text, it will trigger the forgot action

Reply:
Something like this : 
![image](https://github.com/pibooth/pibooth/assets/13900677/713a1775-24dc-4f4d-8477-4d57966a924b)

Reply:
unfrotunately we cannot move the text to the left as the picture in portrait mode is taking the full height, that's the main reason we kept it on the right. The best workaround we gave was to update the text to be more explicit like "Appuyer sur la photo pour l'oublier" but we cannot make it default as the pibooth can be used without a touchscreen and in this case the arrow is showing which button should be pressed.

Reply:
Something like this, with the text below the left image and the printer centered
![image](https://github.com/pibooth/pibooth/assets/13900677/ed11c0c7-b823-4343-8201-c81fe33be110)

And in portrait mode, the image is slowly reduced to show the text below (only in touchscreen mode)
![image](https://github.com/pibooth/pibooth/assets/13900677/660b3b31-c97e-4c10-a45d-2538f84a8c69)

I've added a test if we are in portrait mode and touchscreen mode, in this case I align on top and reduce to 80% the image. What do you think about this solution ?